<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrfStage2Dialog</class>
 <widget class="QDialog" name="PrfStage2Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QPrf - Stage 2 Stimuli Generation</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Basic</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Input file name (.mat):</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="3">
        <widget class="QLineEdit" name="varname_lineEdit">
         <property name="text">
          <string>stimulus{0,0}</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="browseFile_pushButton">
         <property name="text">
          <string>&amp;Browse...</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="5">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Stimulus variable must be a an array of size [H, W, TOT_DUR], where: H - stimulus height, W - stimulus width, TOT_DUR - total stimulus duration.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Stimulus variable name:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="3">
        <widget class="QLineEdit" name="fileName_lineEdit"/>
       </item>
       <item row="5" column="0" colspan="5">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="0" colspan="3">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Please refer to Advanced tab for further options.</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QPushButton" name="browseVar_pushButton">
         <property name="text">
          <string>B&amp;rowse...</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_15">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Output file name (.mat):</string>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QPushButton" name="browseOut_pushButton">
         <property name="text">
          <string>Br&amp;owse...</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="5">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>By default, stage 2 file will be saved to the same directory as input file.</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="3">
        <widget class="QLineEdit" name="outName_lineEdit">
         <property name="text">
          <string>stage2.mat</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="5" column="2">
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>RF Size Step:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="3">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0">
        <widget class="QLineEdit" name="yStart_lineEdit">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_9">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>X End:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="xEnd_lineEdit">
         <property name="text">
          <string>101</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Y Step:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_8">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>X Start:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_13">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>RF Size Start:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="xStep_lineEdit">
         <property name="text">
          <string>4</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>X Step:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_11">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Y Start:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="yEnd_lineEdit">
         <property name="text">
          <string>101</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLineEdit" name="yStep_lineEdit">
         <property name="text">
          <string>4</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="label_12">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>RF Size End:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLineEdit" name="rfsizeStart_lineEdit">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLineEdit" name="xStart_lineEdit">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="rfsizeEnd_lineEdit">
         <property name="text">
          <string>101</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="label_10">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Y End:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QLineEdit" name="rfsizeStep_lineEdit">
         <property name="text">
          <string>4</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Specify ranges for precomputed stage 2 stimulus parameters (X, Y, RF Size).</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>xStart_lineEdit</tabstop>
  <tabstop>xEnd_lineEdit</tabstop>
  <tabstop>xStep_lineEdit</tabstop>
  <tabstop>yStart_lineEdit</tabstop>
  <tabstop>yEnd_lineEdit</tabstop>
  <tabstop>yStep_lineEdit</tabstop>
  <tabstop>rfsizeStart_lineEdit</tabstop>
  <tabstop>rfsizeEnd_lineEdit</tabstop>
  <tabstop>rfsizeStep_lineEdit</tabstop>
  <tabstop>fileName_lineEdit</tabstop>
  <tabstop>browseFile_pushButton</tabstop>
  <tabstop>varname_lineEdit</tabstop>
  <tabstop>browseVar_pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PrfStage2Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PrfStage2Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
